Spigot-API-Patches/0228-EntityMoveEvent.patch

```diff
@@ -0,0 +1,107 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: William Blake Galbreath <Blake.Galbreath@GMail.com>
+Date: Tue, 11 Feb 2020 21:56:38 -0600
+Subject: [PATCH] EntityMoveEvent
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/event/entity/EntityMoveEvent.java b/src/main/java/com/destroystokyo/paper/event/entity/EntityMoveEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..87fb271141a9604d879455f029d5cc009637afe3
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/event/entity/EntityMoveEvent.java
+@@ -0,0 +1,95 @@
++package com.destroystokyo.paper.event.entity;
```

Comment:
```suggestion
+package io.papermc.paper.event.entity;
```

Reply:
I thought the plan was to move everything to the new package all at once so we don't split it up and create confusion

Reply:
That'll only be more work later on. We've already got some on the new package, and they're not moving in that case; little to lose by just using the new one at this point.